docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,146 @@
+# Quickstart
+Learn the basics by setting up and interacting with the [Ceramic Javascript Client](./javascript/installation.md).
+
+!!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic Components in the browser.
+
+## **1. Install the Client**
+
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
+
+## **2. Create a stream**
+Creating streams is dependent on the StreamType. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample using the [TileDocument StreamType](../streamtypes/tile-document/overview.md) can be found below.
+
+=== "Command"
+
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+    const schemaId = 'k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw' // this is our Basic Profile Schema.
+    
+    const content = {
+      name: 'Your Name',
+      emoji: '✌🏻',
+      description: 'Something about yourself'
+    }
+    const metadata = {
+      controllers: [ceramic.did.id],
+      family: 'Basic Profile',
+      schema: schemaId
+    }
+
+    const doc = await TileDocument.create(ceramic, content, metadata)
+
+    const streamId = doc.id.toString()
+    ```
+
+## **3. Query a stream**
+Use the `load()` function to query the current [state](../learn/glossary.md#state) of a stream. You will need to provide it's *StreamID*.
+
+=== "Command"
+
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
```

Comment:
```suggestion
    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"  // This is just an example StreamID, you will need to use a StreamID relevant to your app.
```

Reply:
This shouldn't be necessary but unfortunately we've already gotten people trying to follow our examples **exactly** and hitting issues. :facepalm:

Reply:
This actually is a live stream and not a demo stream. All demo streams are concatenated so that they are not useable specifically for this reason!